test/views/ShutMucrSummaryViewSpec.scala

```diff
@@ -65,7 +65,7 @@ class ShutMucrSummaryViewSpec extends UnitViewSpec with CommonMessages {
       val changeButton = view.getElementById("shutMucr-change")
 
       changeButton must haveHref(routes.ShutMucrController.displayPage())
-      changeButton.text() mustBe messages("site.edit")
+      changeButton.text() contains messages("site.edit")
```

Comment:
Are you sure about `contains`?

Reply:
maybe better include - there was quite different behaves in scalatest about those matchers

Reply:
yes good point. Done.